src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,8 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            internal FixedString.Size260 cFileName;
+            internal FixedString.Size14 cAlternateFileName;
```

Comment:
> Well, you introduce that problem with any solution that doesn't allow the marshaller to convert it to a string for you.

A property off of a field like this hides it, and makes it less obvious when writing code like:
https://github.com/dotnet/corefx/pull/22596/files#diff-21c998b44d4d2b9bff878c45a7e1fb65R402
(even though that debug, it highlights how non-obvious it is that each of those Value calls is creating a new string)